assets/oit_demo/shaders/Composite.hlsl

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#define IS_SHADER
+#include "Common.hlsli"
+#include "FullscreenVS.hlsli"
+
+Texture2D    OpaqueTexture       : register(OPAQUE_TEXTURE_REGISTER);
+Texture2D    TransparencyTexture : register(TRANSPARENCY_TEXTURE_REGISTER);
+
+float4 psmain(VSOutput input) : SV_TARGET
+{
+    const int3 texelCoord = int3(input.position.xy, 0);
+
+    const float3 opaqueColor = OpaqueTexture.Load(texelCoord).rgb;
```

Comment:
This causes validation errors on Vulkan:
 - this load requires StorageImageReadWithoutFormat,
 - the shader pipeline is build without this capability enabled.
 Not sure how to do that for BigWheels. Not sure of the implications to always enable this capability?

Reply:
I don't get that error in Vulkan (I mostly develop in Vulkan with debug at true).
I use 2 different AMD GPUs: WX3200 and RX6700XT.
Maybe my driver is more permissive than yours?

It makes sense that it happens though.
I changed that **Load** into a regular **Sample**.
I believe it should fix the issue on your driver.
Please let me know if you encounter more of those.

Reply:
Ah yes, that's better, no need to add the capability. Btw, all Loads will generate this capability, so the others also need patching.

I can reproduce on my workstation (WX 3200), not if I use the AMD gpu, but if I use LLVMpipe.
(forced the gpu index to 1, which is llvmpipe for me).

Reply:
Ah I see!
I thought it was related to the image format (uint while others are float), but it is more of a software driver limitation.
I have another PR coming soon.
I will remove all loads from it, as sampling with a nearest sampler just works fine.